Makefile

```diff
@@ -3,7 +3,7 @@ lib: es6-lib node_modules
 	node_modules/.bin/babel es6-lib --out-dir lib
 
 test: lib
-	GEO_IMPORT_ENV=test ZOOKEEPER_ENSEMBLE="{}" ./node_modules/.bin/mocha es6-test/unit es6-test/smoke --require @babel/register && jshint es6-lib
+	GEO_IMPORT_ENV=test ./node_modules/.bin/mocha es6-test/unit es6-test/smoke --require @babel/register && jshint es6-lib
```

Comment:
Why  does ZOOKEEPER_ENSEMBLE no longer have to be specified?

Reply:
It doesn't have to be specified because we no longer throw errors when `prod.sh` is imported in `es6-lib/config/index.js`.